src/app/clusters/energy-evse-server/energy-evse-server.h

```diff
@@ -81,6 +82,36 @@ class Delegate
      */
     virtual Protocols::InteractionModel::Status StartDiagnostics() = 0;
 
+    /**
+     * @brief Delegate should implement a handler for the SetTargets command.
+     * It should report Status::Success if successful and may
+     * return other Status codes if it fails
+     */
+    virtual Protocols::InteractionModel::Status
+    SetTargets(const DataModel::DecodableList<Structs::ChargingTargetScheduleStruct::DecodableType> & chargingTargetSchedules) = 0;
+
+    /**
+     * @brief Delegate should implement a handler for LoadTargets
+     *
+     * This needs to load any stored targets into memory
+     */
+    virtual Protocols::InteractionModel::Status LoadTargets() = 0;
```

Comment:
why are these returning Status instead of CHIP_ERROR?

Reply:
One reason to return Status is to emphasize that you have to return a spec-defined status.